model/atmosphere/diffusion/src/icon4py/model/atmosphere/diffusion/diffusion.py

```diff
@@ -406,13 +406,14 @@ def __init__(
         self.mo_intp_rbf_rbf_vec_interpol_vertex = mo_intp_rbf_rbf_vec_interpol_vertex.with_backend(
             self._backend
         ).compile(
+            enable_jit=False,
```

Comment:
the problem with this approach is that we don't check programs that are missing a compile, but probably ok for now. If we would have context manager  + context var support for the config, we could just switch the default when executing.